src/routes.js

```diff
@@ -1,2 +1,7 @@
 export const HOME_ROUTE = 'home';
 export const CONTACT_ROUTE = 'contact';
+export const ABOUT_ROUTE = 'about';
+export const QUESTIONS_ROUTE = 'questions';
+export const QUESTION_PLUS_ROUTE = 'question+';
+export const SIGNUP_ROUTE = 'signup';
+export const SIGNIN_ROUTE = 'signin';
```

Comment:
```suggestion
export const HOME_ROUTE = '/';
export const CONTACT_ROUTE = '/contact';
export const ABOUT_ROUTE = '/about';
export const QUESTIONS_ROUTE = '/questions';
export const QUESTION_PLUS_ROUTE = '/question+'; 
export const SIGNUP_ROUTE = '/signup';
export const SIGNIN_ROUTE = '/signin';
```

Reply:
I am not sure about the questions+ route looks weird, what are you trying to do exactly?